cloud/spawn.go

```diff
@@ -324,15 +324,10 @@ func ModifySpawnHost(ctx context.Context, env evergreen.Environment, host *host.
 func MakeExtendedSpawnHostExpiration(host *host.Host, extendBy time.Duration) (time.Time, error) {
 	if err := host.PastMaxExpiration(extendBy); err != nil {
 		return time.Time{}, err
+	} else {
+		newExp := host.ExpirationTime.Add(extendBy)
+		return newExp, nil
 	}
-
-	newExp := host.ExpirationTime.Add(extendBy)
-	remainingDuration := newExp.Sub(time.Now()) //nolint
```

Comment:
should we maybe move this check above `host.ExpirationTime.Add` instead of removing it?